airflow/executors/kubernetes_executor.py

```diff
@@ -302,7 +302,6 @@ def run_next(self, next_job: KubernetesJobType) -> None:
         status
         """
         key, command, kube_executor_config, pod_template_file = next_job
-        self.log.info('Kubernetes job is %s', key)
```

Comment:
This was added by @dstandish so perhaps we can get some context

Reply:
alas, i just moved it
and @collinmcnulty touched it before me, and it was added by fokko ...5 years ago!
apparently i also got rid of the one-lining logic that @collinmcnulty added
in any case, probably there's no one better to review this PR than @collinmcnulty

Reply:
maybe he'll want us to keep it and reintroduce the one-lining

Reply:
This log is not really removed, you can say i moved it to line 335 to include the pod-name details in it. So unless anything fails on line 306-335 i believe it is ok. This path doesn't involve any external communication so all information relevant to an error would be hopefully be included in exceptions if any.

Reply:
I like the new line 335. LGTM